keys/keys.go

```diff
@@ -0,0 +1,87 @@
+package keys
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"reflect"
+
+	"github.com/open-policy-agent/opa/util"
+)
+
+const (
+	// DefaultTokenSigningAlg Algorithm to use if not provided
+	DefaultTokenSigningAlg = "RS256"
+)
+
+// KeyConfig holds the keys used to sign or verify bundles and tokens
+type KeyConfig struct {
```

Comment:
How would an EC256 key look using this format? Reason I ask is that I prefer the RFC JWK format (https://tools.ietf.org/html/rfc7517#section-3). This is also input to io.jwt.verify_<alg>

From tokens_test.go (only public keys, but you get the idea.)
```
{
    "keys": [
        {
          "kty": "EC",
          "use": "sig",
          "crv": "P-256",
          "kid": "k1",
          "x": "9Qq5S5VqMQoH-FOI4atcH6V3bua03C-5ZMZMG1rszwA",
          "y": "LLbFxWkGBEBrTm1GMYZJy1OXCH1KLweJMCgIEPIsibU",
          "alg": "ES256"
        },
        {
          "kty": "RSA",
          "e": "AQAB",
          "use": "enc",
          "kid": "k2",
          "alg": "RS256",
          "n": "sGu-fYVE2nq2dPxJlqAMI0Z8G3FD0XcWDnD8mkfO1ddKRGuUQZmfj4gWeZGyIk3cnuoy7KJCEqa3daXc08QHuFZyfn0rH33t8_AFsvb0q0i7R2FK-Gdqs_E0-sGpYMsRJdZWfCioLkYjIHEuVnRbi3DEsWqe484rEGbKF60jNRgGC4b-8pz-E538ZkssWxcqHrYIj5bjGEU36onjS3M_yrTuNvzv_8wRioK4fbcwmGne9bDxu8LcoSReWpPn0CnUkWnfqroRcMJnC87ZuJagDW1ZWCmU3psdsVanmFFh0DP6z0fsA4h8G2n9-qp-LEKFaWwo3IWlOsIzU3MHdcEiGw"
        }
	]
}
```

Reply:
Oh yeah I did not take `EC*` algorithms into account here as I used GCP as my test target (where only RSA is supported), but you are absolutely right that I should. 

As for providing JWKs in conf that would be a nice addition, but perhaps somewhat out of scope for this PR.

Reply:
Now done.